worldedit-core/src/main/java/com/fastasyncworldedit/core/configuration/Settings.java

```diff
@@ -289,8 +289,18 @@ public static final class ENABLED_COMPONENTS {
         @Comment({"Show additional information in console. It helps us at IntellectualSites to find out more about an issue.",
                 "Leave it off if you don't need it, it can spam your console."})
         public boolean DEBUG = false;
-        @Comment({"Whether or not FAWE should notify you on startup about new versions available."})
-        public boolean UPDATE_NOTIFICATIONS = true;
+
+        @Migrate("enabled-components.update-notification")
+        @Comment({"Whether or not FAWE should notify you on startup about new available snapshots."})
+        public boolean SNAPSHOT_UPDATE_NOTIFICATIONS = true;
```

Comment:
Should this be false by default?

Reply:
I did not want to change the current behavior by default tbh. I think it's reasonable to keep notifications/checks for new Jenkins builds (especially because of bug fixes etc). In the end anyone can simply disable that themselves ig.

But I haven't spent much time thinking about it, so I'm open for input.